stores/sql/mysql/main.go

```diff
@@ -1087,6 +1087,31 @@ func (tx *MainDatabaseTx) UpdateHostBlocklistEntries(ctx context.Context, add, r
 				return fmt.Errorf("failed to join host blocklist entry: %w", err)
 			}
 		}
+
+		v2JoinStmt, err := tx.Prepare(ctx, `
+		INSERT IGNORE INTO host_blocklist_entry_hosts (db_blocklist_entry_id, db_host_id)
+		SELECT ?, db_host_id FROM (
+			SELECT db_host_id
```

Comment:
I think you might be able to just extract this inner block and attach it to the query above with the `UNION ALL` operator.